ghost/admin/app/components/gh-psm-tags-input.js

```diff
@@ -13,6 +13,8 @@ export default class GhPsmTagsInput extends Component {
     // public attrs
     post = null;
 
+    currentTags = null;
```

Comment:
What the reasoning behind adding this local state/indirection? Generally it's always preferable to work directly with model state rather than duplicating it across internal component state.

Reply:
Thanks, addressed!